sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosAuthorizationTokenResolver.java

```diff
@@ -1,33 +1,33 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.cosmos.implementation;
+package com.azure.cosmos.models;
```

Comment:
who is the customer for this new public API?

as the future path of authentication for many customers is AAD why can't we direct the customer to use AAD?